app/react/App/ErrorHandling/ErrorUtils.ts

```diff
@@ -0,0 +1,29 @@
+export interface RequestError extends Error {
+  title?: string;
+  summary?: string;
+  requestId?: string;
+  code?: string;
+}
+
+export interface APIError {
+  json: {
+    error?: string;
+    prettyMessage: string;
+    requestId?: number;
+  };
+  status?: number;
+}
+
+export const getRenderError = (apiResponse: APIError) => {
```

Comment:
@mfacar do not understand the name of the function, what is this supposed to do ? normalize the error ?

Reply:
Yes, it adapt the object to the needed by the UI, I changed the name